src/core/qgis.h

```diff
@@ -330,6 +330,41 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( VectorExportResult )
 
+    /**
+     * SqlLayerDefinitionCapability enum lists the arguments supported by the provider when creating SQL query layers.
+     * \since QGIS 3.22
+     */
+    enum class SqlLayerDefinitionCapability : int
+    {
+      Filter = 1 << 1,            //!< SQL layer definition supports filter
```

Comment:
What exactly does "filter" mean? Is it referring to vector layer subset strings?

Reply:
Yes, maybe better to rename it to "SubsetStringFilter" ?

Reply:
Much nicer!